genesyscloud/resource_genesyscloud_externalcontacts_contact.go

```diff
@@ -0,0 +1,814 @@
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			"first_name": {
+				Description: "The first name of the contact.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"middle_name": {
+				Description: "The middle name of the contact.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"last_name": {
+				Description: "The middle name of the contact.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"salutation": {
+				Description: "The salutation of the contact.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"title": {
+				Description: "The title of the contact.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"work_phone": {
+				Description: "Contact work phone settings.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Computed:    true,
+				Elem:        phoneNumber,
+			},
+			"cell_phone": {
+				Description: "Contact call phone settings.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        phoneNumber,
+			},
+			"home_phone": {
+				Description: "Contact home phone settings.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        phoneNumber,
+			},
+			"other_phone": {
+				Description: "Contact other phone settings.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        phoneNumber,
+			},
+			"work_email": {
+				Description: "Contact work email.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"personnal_email": {
+				Description: "Contact personnal email.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"other_email": {
+				Description: "Contact other email.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"address": {
+				Description: "Contact address.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        address,
+			},
+			"twitter_id": {
+				Description: "Contact twitter account informations.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        twitterId,
+			},
+			"line_id": {
+				Description: "Contact line account informations.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        lineId,
+			},
+			"whatsapp_id": {
+				Description: "Contact whatsapp account informations.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Computed:    true,
+				Elem:        whatsappId,
+			},
+			"facebook_id": {
+				Description: "Contact facebook account informations.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        facebookId,
+			},
+			"external_organization": {
```

Comment:
Is this meant to have a different description than `survey_opt_out` ? It might be better to change the name to `external_organization_id`  
Also I don't see this field included in the create/update or read functions

Reply:
Same description is an error.
But external organization is not implemented, so I dont know if its better to remove this field until external orgs is implemented ?

Reply:
I choose to remove the external_organization field until External Orgs is implemented (im working on it).